src/main/resources/static/index.html

```diff
@@ -0,0 +1 @@
+<meta http-equiv="refresh" content="0; URL='/game-list/platform'">
```

Comment:
I added this snip-it of code that we learned in class so when the user just types '127.0.0.1:8080' into the address bar, it reroutes to the platforms page - making it the home page / landing page.